CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+cmake_minimum_required(VERSION 3.19.0)
+
+project(NCOM2CSV)
+
+if(${CMAKE_SYSTEM_NAME} MATCHES Darwin)
+    message("Apple OS (Darwin) detected")
+elseif(${CMAKE_SYSTEM_NAME} MATCHES Linux)
+    message("Linux OS detected")
+else()
+    message(FATAL_ERROR "${CMAKE_SYSTEM_NAME} OS detected - currently not supported - EXIT")
+endif()
+
+include_directories(nav)
```

Comment:
I suggest using `target_include_directories()` instead.

Reply:
ok.
would lead to something like this, or?
cmake_minimum_required(VERSION 3.19.0)

project(NCOM2CSV)

if(${CMAKE_SYSTEM_NAME} MATCHES Darwin)
    message("Apple OS (Darwin) detected")
elseif(${CMAKE_SYSTEM_NAME} MATCHES Linux)
    message("Linux OS detected")
else()
    message(FATAL_ERROR "${CMAKE_SYSTEM_NAME} OS detected - currently not supported - EXIT")
endif()

set(SOURCES nav/NComRxC.c example/NcomToCsv.c)
add_executable(ncom2csv ${SOURCES})
**target_include_directories(ncom2csv PRIVATE nav)**

Reply:
Yes, looks like the way to go.